### PR TITLE
add Shaun Thompson as curator

### PR DESCRIPTION
**Pull request by thaJeztah**
Shaun (@thompson-shaun) is managing the runtime and build teams, and will be helping out on triage, as well as organizing backlogs on GitHub.

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -80,6 +80,7 @@
 			"sam-thibault",
 			"samwhited",
 			"thajeztah"
+			"thompson-shaun",
```
Comment (vvoland) [inline MAINTAINERS:83]:
```suggestion
			"thajeztah",
			"thompson-shaun"
```

**Review by vvoland — CHANGES_REQUESTED** [review]

File: MAINTAINERS (hunk 1 above)
Comment (vvoland) [reply MAINTAINERS:83]: LGTM otherwise 😅

Comment (thaJeztah) [reply MAINTAINERS:83]: doh! I initially had it in the wrong list; let me fix

Comment (thaJeztah) [reply MAINTAINERS]: should be good now

**Review by vvoland — APPROVED** [review]

File: MAINTAINERS [hunk 2]
```diff
@@ -79,7 +79,8 @@
 			"ripcurld",
 			"sam-thibault",
 			"samwhited",
-			"thajeztah"
+			"thajeztah",
+			"thompson-shaun"
```
Comment (thaJeztah) [inline MAINTAINERS:83]: I don't recall; does TOML allow trailing commas in the last entry? If so , I could add one here to reduce code-churn.

Comment (thaJeztah) [reply MAINTAINERS:83]: Looks like it does; so let me update once more; https://github.com/toml-lang/toml/issues/246#issuecomment-57919189

Comment (vvoland) [reply MAINTAINERS:83]: Looks like it does!

> Arrays can span multiple lines. A terminating comma (also called a trailing comma) is permitted after the last value of the array. 

https://toml.io/en/v1.0.0#array

Comment (thaJeztah) [reply MAINTAINERS]: Yup! I updated to add one, might as well have that in place already 😄 (I didn't update the other list further up yet though as there was no need yet)

**Review by austinvazquez — APPROVED** [review]
Welcome!

**Comment by thaJeztah** [comment]
Let me bring this one in!

**Pull request merged**